plugins/common/shim/config.go

```diff
@@ -82,6 +82,9 @@ func expandEnvVars(contents []byte) string {
 
 func getEnv(key string) string {
 	v := os.Getenv(key)
+	if v == "" {
```

Comment:
I don't know if it matters for this or not, but os.Getenv() will return an empty string whether the environment variable is set to an empty string or just does not exist. If you need to distinguish these two cases there is os.LookupEnv().

Reply:
I forgot to hit "comment" on my PR about this function heh.

This is a pretty generic function.   Are we expecting ENV vars to always be set?  If someone is driving this off config, rather than env, this will be noisy.

Reply:
> I forgot to hit "comment" on my PR about this function heh.
> 
> This is a pretty generic function. Are we expecting ENV vars to always be set? If someone is driving this off config, rather than env, this will be noisy.

my understanding of the shim is that it's used to expand ENV vars in the config. In this, it will notify if any env var in the config is empty or unset.